src/screens/Settings/BusinessMapping/BusinessMappingEntity.res

```diff
@@ -6,13 +6,14 @@ module ProfileActions = {
   let make = (~defaultProfileName, ~profileId) => {
     open APIUtils
     let getURL = useGetURL()
-    let updateDetails = useUpdateMethod()
+    let updateDetails = useUpdateMethod(~showErrorToast=false)
     let showToast = ToastState.useShowToast()
     let (showModal, setShowModal) = React.useState(_ => false)
     let (businessProfiles, setBusinessProfiles) = Recoil.useRecoilState(
       HyperswitchAtom.businessProfilesAtom,
     )
     let initialValues = [("profile_name", defaultProfileName->JSON.Encode.string)]->Dict.fromArray
+    let fetchProfileList = BusinessProfileHook.useGetProfileList()
```

Comment:
there is a hook useFetchBusinessProfiles, which is getting called at many places and it is setting the business profiles, so i think we might need to call this fetchProfileList at that place also
please verify it doesn't cause any issue somewhere else